CMakeLists.txt

```diff
@@ -208,7 +209,8 @@ if(RADIUM_ENABLE_TESTING)
     # prepare integration tests
     if(NOT RADIUM_ENABLE_EXAMPLES)
         add_subdirectory(examples/CoreExample EXCLUDE_FROM_ALL)
-        if(RADIUM_GENERATE_LIB_HEADLESS)
+        find_package(Radium COMPONENTS Headless QUIET)
```

Comment:
Why doing a `find_package` here as we have means to test for the presence of Headless component without side effect.

Reply:
well, it's another proposal, I've kept most of you proposal in //
My proposal is not to change RADIUM_GENERATE_LIB_HEADLESS ... but I let you decide what you think is best.
If it's to change RADIUM_GENERATE_LIB_HEADLESS, then you can also remove RADIUM_COMPONENTS related stuff, since they are redudant

Reply:
I dont really care, simple difference of opinion, except for the side effect I removed in the next commit. I also make final message less disturbing printing missing components only if they are some.